### PR TITLE
Ticket8210 add ioc lndyisw

### PR DESCRIPTION
**Pull request by Chsudeepta**
IOC created for LYNDY ISwitch device.
This is part of the [ticket](https://github.com/ISISComputingGroup/IBEX/issues/8210)

### CODE REVIEW

**Review by LowriJenkins — CHANGES_REQUESTED** [review]
is it not possible to test this ioc at all? (using the test framework/emulator) if so you need to remove the test_that_fails, as it will cause system tests to fail, it looks like none of the test folder is used, might not be nesecary?
Otherwise looks good!

**Comment by Chsudeepta** [comment]
@LowriJenkins . right now we don't have any simulator for the SNMP. I am trying to implement a simulator for SNMP (Freddie has pointed to a library). Once it is found to be working, I intend to update the IOC framework with the test cases for this new IOC

**Comment by Chsudeepta** [comment]
Ok. Then may be we should move it out of review for the time being and see whether the simulator can be implemented.

**Comment by Chsudeepta** [comment]
No. I think you should be able to run the tests after the IOC has been made in the usual manner.

File: system_tests/tests/lndyisw.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            "IPADDR": "127.0.0.1"
+        },
+        "emulator": "Lndyisw",
+        "emulator_launcher_class": CommandLineEmulatorLauncher,
+        "emulator_command_line": "responder.bat",
+    },
+]
+
+
+TEST_MODES = [TestModes.RECSIM, TestModes.DEVSIM]
```
Comment (LowriJenkins) [inline system_tests/tests/lndyisw.py]: I don't think there is any point running in RECSIM since you don't have any simulation records, so can probably just test in DEVSIM

File: system_tests/lewis_emulators/lewis_versions.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+LEWIS_1_3_0 = "1.3.0"
```
Comment (LowriJenkins) [inline system_tests/lewis_emulators/lewis_versions.py]: Can delete this whole folder since we're not using a lewis emulator having this here is just confusing

**Review by LowriJenkins — CHANGES_REQUESTED** [review]
Just some minor changes due to it not using a standard emulator, otherwise looks good

File: system_tests/lewis_emulators/lewis_versions.py (hunk 2 above)
Comment (Chsudeepta) [reply system_tests/lewis_emulators/lewis_versions.py]: This was auto-generated. If the standard practice is to remove then I will go ahead and delete

File: system_tests/tests/lndyisw.py (hunk 1 above)
Comment (Chsudeepta) [reply system_tests/tests/lndyisw.py]: Yes, you are right. We can do away with RECSIM. Removed and checked in

File: system_tests/lewis_emulators/lewis_versions.py (hunk 2 above)
Comment (LowriJenkins) [reply system_tests/lewis_emulators/lewis_versions.py]: I think it might end up being confusing in future if people attempt to emulate this device with an emulator that does nothing, normally an emulator would be set up, but as in this case we are using your simulator instead, it doesn't make sense to keep this

Comment (LowriJenkins) [reply system_tests/lewis_emulators/lewis_versions.py]: Given how rarely we use non-lewis emulators Im not sure we actually have standard practise for this scenario

Comment (Chsudeepta) [reply system_tests/lewis_emulators/lewis_versions.py]: Ok. Deleting the folder then.

**Review by LowriJenkins — APPROVED** [review]
